src/util.ts

```diff
@@ -88,3 +88,26 @@ export function createOutputRetainingCompilerHost(
     outputFiles.set(fileName, content);
   }
 }
+
+/**
+ * Returns the declarations for the given decorator.
```

Comment:
This "util.ts" file is becoming a junk grab-bag.  I think this function belongs in the file that uses it most closely (decorator-annotator).